prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -390,6 +393,19 @@ public async Task SendEnrolleeAbsenceNotificationEmailAsync(int enrolleeId, Enro
 
         private async Task Send(Email email)
         {
+            var doNotEmail = await _context.DoNotEmail
+                .Where(e => String.Equals(e.Email, String.Join(",", email.To), StringComparison.OrdinalIgnoreCase))
+                .Select(e => new {
+                    e.Email,
+                    e.Id
+                })
+                .SingleOrDefaultAsync();
+
+            if (doNotEmail != null) {
+                await _businessEventService.CreateEmailEventAsync("Do not email enrollee");
```

Comment:
Sorry, minor change. Can you change the text in the business event? since the email might not be send to enrolle, "Do not email enrollee" does not make sense. Also, just an idea, to push the to-email-address in the event and indicate that email to this email address has been blocked by do-not-email feature. If someone down the road investigates why an email is not sending out, the business event log can show what email address has been blocked. Hope that makes sense.

Reply:
I think I see what you mean. I have also fixed the case insensitive comparison and was able to extensively test it locally, I will push that up with this change as well.